src/api/v4/client.js

```diff
@@ -506,8 +506,9 @@ function _checkUsername (username) {
 }
 
 function _checkServerUrl (serverUrl, username) {
+  var ipRegex = /^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)/;
```

Comment:
Add the protocol as well.

And I'm sure this regex can be simplified with a group and a quantifier.